README.md

```diff
@@ -101,7 +101,7 @@ If rake fails on your environment, generate the preview [using jekyll](#using-je
 
 1. Press `Ctrl+C` in the serve terminal to stop the server.
 
-> ***TIP***  
+> ***TIP***
```

Comment:
Should it be real tip or note syntax highlighting?

Reply:
Updated!